setup.cfg

```diff
@@ -59,3 +59,4 @@ exclude_lines =
     pragma: no ?cover
     pragma ${PLATFORM_SYSTEM}: no cover
     pragma ${OS_NAME}: no cover
+    pragma Python [^: ]*${PYTHON_VERSION}[^: ]*: no cover
```

Comment:
IMO `[\d.,]*` instead of `[^: ]*` would be more intuitive here to faster understand this line